nat-lab/bin/client

```diff
@@ -11,8 +11,8 @@ sleep 5
 # Add conntrack states to iptables, since running dockers on ubuntu
 # conntrack doesn't find flow entries by default somehow. Works when running dockers on debian tho.
 # ref: https://serverfault.com/a/978715
-iptables -A INPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT
-ip6tables -A INPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT
```

Comment:
Would `RETURN` be enough here?

Reply:
If we use RETURN and later some test wants to add a rule to the INPUT table this rule will never be matched, so I think LOG is better suited in that case

Reply:
`RETURN` target on "top-level" chain would stop the traversal of rules and apply the "policy" (e.g. `iptables -F ....`) target to the packet. Which would mean, that any rules further down the chain will not be evaluated. With `LOG` target, this is non-terminating target, so it is a bit more "noop".